openshift-client/script.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+command="/usr/local/bin/oc-origin"
+
+for args in "$@"; do
```

Comment:
Is this necessary due to a deficiency/bug in array-type param handling? This is pretty gross, and I'd love to not need it.

Reply:
this we are doing for the previous tekton version, we may tag this to use with previous tekton pipeline, and update this without a script for current tekton version i.e. 0.6 as we have provided the feature of array type params.